OrbitGl/TimeGraph.cpp

```diff
@@ -211,6 +211,31 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
+void TimeGraph::HorizontallyScrollIntoView(const TextBox* text_box) {
```

Comment:
"Scroll" in the title suggests an animation to me. Jump or move?

Reply:
Or maybe this could be (with the suggestions below):

```
// Aligns center of textbox with the horizontal position of the view given by 'position' 
// (which defines the position in [0,1] coordinates). 
HorizontallyAlignInView(const TextBox* text_box, double center_position) {...}
```